styles/partials/core/_pages.scss

```diff
@@ -129,39 +132,20 @@
     		width: 586px;
     	}
 
-      #search-box {
-        font-size: 18px;
-        line-height: 30px;
-        float: none;
-        border: 0;
-        box-shadow: none;
-
-        @include tablet{
-          width: calc(100% - 20px);
-          box-shadow: 6px 6px 0px 0px $gray-25;
-          line-height: 37px;
-        }
-        @include desktop{
-          width: calc(100% - 20px);
-          box-shadow: 6px 6px 0px 0px $gray-25;
-          line-height: 37px;
-        }
-      }
-
-      &:hover,
-      &:focus,
-      &:active {
-        background: $search-container-hover;
-        .icon {
-          opacity: 1;
+      .twitter-typeahead {
+        background: $search-container-background;
+        &:hover,
+        &:focus,
+        &:active {
+          background: $search-container-hover;
         }
       }
 
       .icon{
-        height: 51px;
-        width: 55px;
+        height: 50px;
+        width: 50px;
```

Comment:
Ehh kinda a visual regression here — the button should extend over the borders. The production button CSS is a little off  right now anyway so I'll try to fix it as it was originally intended.

Reply:
I think with the border around the typeahead, the old style looks kind of odd. Going to leave it for now.